scripts/eurostat/gdp/us_state_gdp_quarterly_import_config.txt

```diff
@@ -3,7 +3,7 @@ import: {
   provenance_url: "https://ec.europa.eu/eurostat/data/database"
   provenance_description: "Yearly data on GDP in Europe from Eurostat"
   # Path to the resolved MCF. Can be a repeated field.
-  mcf_url: "gs://resolved_mcf/external_tables/eurostatGDPImport13/nodes.mcf-00000-of-00001"
+  mcf_url: "gs://resolved_mcf/external_tables/eurostatGDPImport14/nodes.mcf-00000-of-00001"
```

Comment:
@fpernice-google I just noticed that the title of this file says us_state_gdp. Can you update it to prevent future confusion?

Reply:
Fixed.